odoo/addons/test_http/tests/test_session.py

```diff
@@ -213,3 +217,40 @@ def test_session7_serializable(self):
             with self.assertRaises(TypeError):
                 dict.update(session, foo=value)
             self.assertFalse(session.foo)
+
+
+class TestSessionStore(HttpCaseWithUserDemo):
+    def setUp(self):
+        super().setUp()
+        tmpdir = TemporaryDirectory()
+        config["data_dir"] = tmpdir.name
```

Comment:
the config isn't restored to its original state when the tests are done

Reply:
also I don't think that this configuration is taken into account because `session_store` is a lazy_property, there is a high chance that there already is a session_store cached by the lazy property and that this session_store is still pointing at the *real* store

Reply:
to test that you can simply change the permissions on your real session folder, like to prevent anyone from creating new files (chmod a-w)

Reply:
because you only started your test, start the entire test_http suite and you'll see ;)